lib/urlwatch/storage.py

```diff
@@ -126,6 +126,8 @@
             'enabled': False,
             'bot_token': '',
             'chat_id': '',
+            'monospaced': False,
+            'disable_notification': False,
```

Comment:
Use `silent` instead of `disable_notification`.

Reply:
renamed